src/umessage/umessagebuilder.rs

```diff
@@ -14,16 +14,16 @@
 use bytes::Bytes;
 use protobuf::Message;
 
-use crate::transport::validator::UAttributesValidator;
-use crate::uprotocol::{
-    Data, UAttributes, UMessage, UMessageType, UPayload, UPayloadFormat, UPriority, UUri, UUID,
+use crate::errors::ValidationError;
```

Comment:
FMPOV this should be `use crate::uattributes::uattributesvalidator::ValidationError` so that we use the error type that the validator has defined in its module. Currently, that is indeed `errors::ValidationError` but this is an implementation detail of the validator.

Reply:
Ok